index.d.ts

```diff
@@ -38,4 +38,44 @@ export default function pFilter<ValueType>(
 	options?: Options
 ): Promise<ValueType[]>;
 
+/**
+Filter promises concurrently.
+
+@param input - Iterated over concurrently in the `filterer` function.
+@param filterer - The filterer function that decides whether an element should be included into result.
+@param options - See the [`p-map` options](https://github.com/sindresorhus/p-map#options).
+@returns An async iterable that streams each value from `input` for which the `filterer` function returned `true`.
+
+@example
+```
+import {pFilterIterable} from 'p-map';
+import getWeather from 'get-weather'; // Not a real module
+
+const places = [
+	getCapital('Norway').then(info => info.name),
+	'Bangkok, Thailand',
+	'Berlin, Germany',
+	'Tokyo, Japan',
+];
+
+const filterer = async place => {
+	const weather = await getWeather(place);
+	return weather.temperature > 30;
+};
+
+const result = await pFilterIterable(places, filterer);
+
+console.log(result);
+//=> ['Bangkok, Thailand']
+```
```

Comment:
The example is not correct. And the example should also be in the readme under the `pFilterIterable` section.

Reply:
fixed 👍🏻